src/main/java/com/fasterxml/jackson/core/util/BufferRecyclers.java

```diff
@@ -185,66 +185,4 @@ public static void quoteAsJsonText(CharSequence input, StringBuilder output) {
     public static byte[] quoteAsJsonUTF8(String rawText) {
         return JsonStringEncoder.getInstance().quoteAsUTF8(rawText);
     }
-
-    /*
-    /**********************************************************************
-    /* Default BufferRecyclerPool implementations
-    /**********************************************************************
-     */
-
-    public static BufferRecyclerPool defaultRecyclerPool() {
-        return ThreadLocalRecyclerPool.INSTANCE;
-    }
-
-    public static BufferRecyclerPool nopRecyclerPool() {
-        return NonRecyclingRecyclerPool.INSTANCE;
-    }
-
-    /**
-     * Default {@link BufferRecyclerPool} implementation that uses
-     * {@link ThreadLocal} for recycling instances.
-     *
-     * @since 2.16
-     */
-    public static class ThreadLocalRecyclerPool
```

Comment:
Is this change agreed with @cowtowncoder ? Also, is there agreement to take the TokenStreamFactory forFactory out of the acquireBufferRecycler params?

Reply:
Let me think about this a bit. I don't mind moving implementations out per se, but not sure I like `Strategy` approach.

Reply:
Ok, so removal of Strategy made sense to me.

And now that I think it through, removal of `TokenStreamFactory` does make sense: it would be useful for constructing or initializing pool, but not for every recycler access.

I am tempted to add some sort of "init(TokenStreamFactory)" method in `BufferRecyclerPool`, but right now don't really have compelling reason for that -- theoretically this would be useful for letting Pool implementations access `TokenStreamFactory` configuration, but none of implementations do that.
And I think if an implementation wants that, they can require use of factory method, and/or just expose configurability explicitly.

So I think this is fine.